packages/sirius-web/backend/sirius-web-papaya/src/main/java/org/eclipse/sirius/web/papaya/services/PapayaRepresentationProvider.java

```diff
@@ -0,0 +1,25 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.web.papaya.services;
+
+import org.eclipse.sirius.components.core.api.IEditingContextProcessor;
+import org.springframework.stereotype.Service;
+
+/**
+ * Used to register the various representations which will be contributed in a papaya project.
+ *
+ * @author sbegaudeau
+ */
+@Service
+public class PapayaRepresentationProvider implements IEditingContextProcessor {
+}
```

Comment:
What is the purpose of this class, is it the same as `PapayaEditingContextInitializer`?

Reply:
It's really useless indeed, I've removed it :)